schema/annotation-source.yaml

```diff
@@ -25,22 +25,23 @@ $defs:
         type: string
         description: >-
           A free-text description of the InformationEntity.
-      strength:
+      confidence_level:
         $ref_curie: gks.core:Coding
         description: >-
-          A coded term describing the strength of support that the information the statement
-          represents is true.
+          A term describing the degree of confidence held by the creator 
+          of the information entity, that the information it represents is true 
+          (e.g. 'high confidence', 'likely true')
       confidence_score:
```

Comment:
What is the use case for this attribute . . .if intended specifically to hold quantitative scores reflecting the strength of evidence supporting a Statement - I would use the 'Statement.evidence_score' attribute I propose further down.

Reply:
it is really the "result" of a method. The final score for the method used to weigh the evidence and any other expert input. I would think of this as the "result" section in any scientific paper or clinical lab result. It is the overall score (if it can be quantified) and usually goes hand-in-hand with a qualified term for human consumption. 
For example LDL score is 150 and is Above Normal Range.

Reply:
The `InformationEntity.confidence_score` attribute in the VA Core model is simply meant to be a quantitative version of the qualitative `confidence_level` attribute . . . a numerical score that reflects the accuracy/truthfulness of any given information entity (be it a data item, image, statement, etc.).  It gets inherited by descendant classes including Statement - where it can be used specifically to record scores indicating a confidence in the truth of a target proposition.

Larry what you are describing with your examples above seems to me to be a different thing, but something the VA model is certainly capable of handling. I think we can clear things up pretty quickly when we talk next.